src/components/reusable/sideDrawer/sideDrawer.scss

```diff
@@ -174,15 +164,18 @@ h1 {
       transform: none;
 
       &::backdrop {
-        background-color: rgba(0, 0, 0, 0.4);
+        background: var(--kd-color-background-opacity-4);
+        backdrop-filter: blur(calc(var(--kd-blur-medium, 4px) / 2));
```

Comment:
This should really just be a token without any calc.

Reply:
naming this `--kd-background-blur-medium` to be added as a manual token